### DISCUSSION TITLE
WebdriverIO exceptions or errors returned in Console output mostly not help full at all to debug or fix your code

### DISCUSSION THREAD

**Discussion by anupamtomar** [webdriverio/webdriverio#10289] [General]
**WebdriveIO version we are using - v8.7.0**

**Please help me if I made any mistake or if WebdriverIO itself has not handled such errors.**

Now what user can do with this piece of information in the console? The user can't even figure out which line of your code or element has an issue with the below error info-

```
element not interactable: element not interactable
  (Session info: chrome=112.0.5615.138)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

```
[chrome 112.0.5615.138 windows #0-0] element not interactable
  (Session info: chrome=112.0.5615.138)
[chrome 112.0.5615.138 windows #0-0] element not interactable: element not interactable
[chrome 112.0.5615.138 windows #0-0]   (Session info: chrome=112.0.5615.138)
[chrome 112.0.5615.138 windows #0-0]     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by christian-bromann** [comment]
@anupamtomar do you have a reproducible example when the stack trace doesn't provide helpful information?

**Reply by anupamtomar** [reply]
yes! Actually, I was getting this error when the element was not in the viewable area of the page, it was resolved by using `element.scrollIntoView();`

**Comment by anupamtomar** [comment]
Are we planning to resolve such stack trace issues in any future releases? or it gets resolved already in any release? because still there are many instances where I have faced such issues mostly with `element not interactable: element not interactable`

**Reply by christian-bromann** [reply]
It would be desirable to have better error messages. We would appreciate any help on this, thank you!

**Comment by zbyte64** [comment]
Experiencing something similar after upgrading from webdriverio 6:

```
TypeError: Cannot read properties of undefined (reading 'options')
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Not currently sure what is causing the error, but I would like to understand how to better debug this (without injecting console log statements line by line preferably).
